core/kazoo_documents/src/kzd_limits.erl.src

```diff
@@ -74,6 +75,18 @@ inbound_trunks(Doc, Default) ->
 set_inbound_trunks(Doc, InboundTrunks) ->
     kz_json:set_value([<<"inbound_trunks">>], InboundTrunks, Doc).
 
+-spec onnet_exempt(doc()) -> boolean().
```

Comment:
Similar to the `ref` vs `doc` situation, KAZOO auto-generates these `.erl.src` files from the schemas. Any changes here should be migrated to the `.erl` version and committed.